web/client/src/components/form/TokenForm.jsx

```diff
@@ -304,10 +295,10 @@ export function FormikStepper(props) {
     <Formik
       {...props}
       validationSchema={currentChild.props.validationSchema}
-      onSubmit={async (values, helpers) => {
+      onSubmit={async (values) => {
         if (isLastStep()) {
-          await props.onSubmit(values, helpers);
           setDone(true);
+          await props.onSubmit(values);
```

Comment:
Because the helper functions are no longer being used in the getToken function since it was unnecessary. Instead, onReset is used to call handleReset which will reset the form when  the reset button is clicked.